src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -89,4 +89,17 @@ public static boolean isNonZeroUnsignedInteger(String s) {
             return false;
         }
     }
+
+    /**
+     * Trims whitesapce on both inside and outside the string. E.g. "  Tom     Dick " -> "Tom Dick"
+     * @param input Input to trim
+     * @return Trimmed input
+     */
+    public static String trimWhitespace(String input) {
+        // Remove leading and trailing whitespace
```

Comment:
Khang Hou, do you think we need a `null` check here via the `requireAllNotNull()`?

Reply:
Thanks for pointing this out. Done